lti/lti-common/src/java/org/sakaiproject/lti13/LineItemUtil.java

```diff
@@ -324,7 +324,10 @@ public static Map<String, String> getExternalIdsForToolAssignments(String contex
 				retval.put(assignmentReference, constructExternalId(content, null));
 			}
 		} catch (Throwable e) {
-			log.error("Unexpected Throwable", e.getMessage());
+			log.error("Unexpected Throwable", e.toString());
+			if ( log.isDebugEnabled() ) {
+				e.printStackTrace();
+			}
```

Comment:
you don't need to use `log.isDebugEnabled()`

```suggestion
			log.error("Unexpected Throwable", e.toString());
			log.debug("Stacktrace", e);
```

Reply:
Thanks - I really like this pattern.